phpstan-baseline.neon

```diff
@@ -0,0 +1,82 @@
+parameters:
+	ignoreErrors:
```

Comment:
this file contains all errors phpstan currently reports. because of they are contained in the baseline phpstan will ignore those when invoking `composer phpstan`.

we should fix the errors in a separate PR.